src/OrchardCore.Modules/OrchardCore.Users/Controllers/TwoFactorAuthenticationController.cs

```diff
@@ -27,6 +28,8 @@ namespace OrchardCore.Users.Controllers;
 [Feature(UserConstants.Features.TwoFactorAuthentication)]
 public class TwoFactorAuthenticationController : TwoFactorAuthenticationBaseController, IUpdateModel
 {
+    public static readonly string ControllerName = typeof(TwoFactorAuthenticationController).ControllerName();
```

Comment:
No need

Reply:
I think it's better to use it here as public so that we can use the same instance anywhere we need it. More helpful than creating a private static instance

Reply:
Why we never use it directly, unless it's used many times

Reply:
I do not want to end up with 100 comments about this change. Undid it.